frontend/apps/marketing/src/selectors/contentful/getImage.ts

```diff
@@ -1,9 +1,23 @@
+import {isWebKitEngine} from '@/selectors/getBrowser';
 import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
 
 export function getRelativeImageUrl(asset: ExperienceAsset | undefined) {
   return asset?.fields?.file?.url;
 }
 
+export function getOptimizedImageFormat(imgUrl: string) {
+  const imgExt = imgUrl.split('.')?.[1]?.toLowerCase();
```

Comment:
I don't think this logic works, for example:

```
"//contentful-images.code.org/90t6bu6vlf76/1ZL5nGgmWBpr80Iv9kDiMG/72c235ebb9319d4f47f73329e85a38e8/hoai_website_hero_right.gif".split('.')[1]
```

This returns `code` which isn't what I'd expect. I think you want to find the last `.` instead?